content/docs/concepts/security/index.md

```diff
@@ -444,6 +444,19 @@ only one authentication policy per mesh and one authentication policy per
 namespace. Istio also requires mesh-wide and namespace-wide policies to have
 the specific name `default`.
 
+If a service has no matching policies, the service behaves as if the following
+`empty-policy` policy applies:
```

Comment:
I don't think this `empty-policy` example is helpful, since it's not even valid. Istio would reject it, because its name must be `default`. Isn't it sufficient to just say that both transport authentication and origin authentication are disabled by default?

Reply:
Yep, agreed. Hopefully the sections on transport and origin auth are clear enough that it is obvious that omitting them both is equivalent to omitting the policy